buildenv/jenkins/testJobTemplate

```diff
@@ -27,6 +27,7 @@ if (!binding.hasVariable('SUFFIX')) SUFFIX = ""
 if (!binding.hasVariable('JCK_GIT_REPO')) JCK_GIT_REPO = ""
 if (!binding.hasVariable('JCK_GIT_REPO_PREFIX')) JCK_GIT_REPO_PREFIX = ""
 if (!binding.hasVariable('SSH_AGENT_CREDENTIAL')) SSH_AGENT_CREDENTIAL = ""
+if (!binding.hasVariable('KEEP_REPORTDIR')) KEEP_REPORTDIR = "true"
```

Comment:
Are the openj9 builds using this testJobTemplate or have they concocted their own?  if they are using it directly, are we changing the default on them?

Just calling it out, in case we need to update settings of openj9 jobs

Otherwise, this LGTM